src/compiler/config/test/validate-config.spec.ts

```diff
@@ -312,6 +312,16 @@ describe('validation', () => {
     expect(validated.diagnostics).toHaveLength(1);
   });
 
+  it('should warn when dist-custom-elements-bundle is found', () => {
+    userConfig.outputTargets = [
+      {
+        type: 'dist-custom-elements-bundle',
+      } as any,
```

Comment:
Do we need this `as any` type assertion? I think this object literal is a [valid Output Target on its own](https://github.com/CODE-AND-MORE/stencil-core-build/blob/9219856097b4f34bfe1125327c675bbef4b5b343/internal/stencil-public-compiler.d.ts#L1740), since all other properties for the `OutputTargetDistCustomElementsBundle` are optional
```
export interface OutputTargetDistCustomElementsBundle extends OutputTargetBaseNext {
  type: 'dist-custom-elements-bundle';
  empty?: boolean;
  externalRuntime?: boolean;
  copy?: CopyTask[];
  inlineDynamicImports?: boolean;
  includeGlobalScripts?: boolean;
  minify?: boolean;
}
// where 'OutputTargetBaseNext' is:
export interface OutputTargetBaseNext {
  type: string;
  dir?: string;
}
```
I'd suggest removing it if we don't need it:
```suggestion
      },
```

Reply:
Nope, I copied from the test above it and missed that. Thanks 👍